runtime/layers/fpm-dev/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG LAYER_IMAGE=bref/php-73:latest
+FROM $LAYER_IMAGE
+
+WORKDIR /opt/bin
+EXPOSE 9000
+
+COPY ./local/php-fpm.conf /opt/bref/etc/php-fpm.conf
+
+CMD [ "php-fpm", "-F", "-y", "/opt/bref/etc/php-fpm.conf" ]
```

Comment:
Posting here so that we don't forget: we should populate the environment variables available on Lambda: https://docs.aws.amazon.com/lambda/latest/dg/lambda-environment-variables.html